### PR TITLE
add ERC20 logic in SC

### LINKED ISSUE #3: add ERC20 logic

### PR DESCRIPTION
**Pull request by lilyanB**
I created 1 function named's: "createPaymentInErc20" and added logic erc20 in the function "pay".

### CODE REVIEW

File: contracts/SilkPayV1.sol [hunk 1]
```diff
@@ -259,27 +412,44 @@ contract SilkPayV1 is Pausable {
...
      * @param _ruling the ruling of disupte
      */
-    function settleFee(PaymentUtils.Payment storage payment, uint256 _ruling) internal {
+    function settleFee(
+        PaymentUtils.Payment storage payment,
+        uint256 _ruling
+    ) internal {
         require(_ruling == SENDER_WINS || _ruling == RECIPIENT_WINS);
         if (_ruling == SENDER_WINS) {
             payable(payment.sender).transfer(payment.amount);
```
Comment (TecSong) [inline contracts/SilkPayV1.sol]: it would also be an ERC20 payment here.

File: contracts/SilkPayV1.sol [hunk 2]
```diff
@@ -299,12 +469,13 @@ contract SilkPayV1 is Pausable {
         payable(payment.sender).transfer(amount);
```
Comment (TecSong) [inline contracts/SilkPayV1.sol]: Same as above.

File: contracts/SilkPayV1.sol
Comment (TecSong) [inline contracts/SilkPayV1.sol:1]: @lilyanB and function pay should add ERC20 logic

**Comment by lilyanB** [comment]
add ERC20 logic on "refound" and "settleFee

**Comment by TecSong** [comment]
and function pay should add ERC20 logic

**Review by TecSong — APPROVED** [review]

**Pull request merged**